ProjFS.Mac/PrjFSKextTests/ProviderMessagingMock.cpp

```diff
@@ -1,12 +1,18 @@
 #include "../PrjFSKext/ProviderMessaging.hpp"
 #include "../PrjFSKext/Locks.hpp"
+#include "../PrjFSKext/VirtualizationRoots.hpp"
 #include "KextLogMock.h"
 #include "KextMockUtilities.hpp"
 
 #include <sys/proc.h>
 #include <libkern/OSAtomic.h>
 #include <sys/kauth.h>
+#include "ProviderMessagingMock.hpp"
 
+bool static s_requestResult = true;
+bool static s_secondRequestResult = false;
+bool static s_cleanupRoots = false;
```

Comment:
Minor: Something like `s_cleanupRootsAfterRequest` might be a more descriptive name.